guides/md/sequential_model.md

```diff
@@ -540,6 +540,5 @@ That's about all you need to know about Sequential models!
 
 To find out more about building models in Keras, see:
 
-- [Guide to the Functional API](/guides/functional_api/)
-- [Guide to making new Layers & Models via subclassing](
-    /guides/making_new_layers_and_models_via_subclassing/)
+- [Guide to the Functional API](https://github.com/keras-team/keras-io/blob/master/guides/functional_api.py)
```

Comment:
Use the keras.io links instead

Reply:
@fchollet 
As per suggestion I updated the keras.io links.
Thanks !